libraries/AP_Scheduler/PerfInfo.cpp

```diff
@@ -114,13 +115,14 @@ float AP::PerfInfo::get_filtered_time() const
 void AP::PerfInfo::update_logging()
 {
     gcs().send_text(MAV_SEVERITY_WARNING,
-                    "PERF: %u/%u max=%lu min=%lu F=%u sd=%lu",
+                    "PERF: %u/%u [%lu:%lu] F=%u sd=%lu Ex=%lu",
                     (unsigned)get_num_long_running(),
                     (unsigned)get_num_loops(),
                     (unsigned long)get_max_time(),
                     (unsigned long)get_min_time(),
                     (unsigned)(get_filtered_time()*1.0e6),
-                    (unsigned long)get_stddev_time());
+                    (unsigned long)get_stddev_time(),
+                    AP::scheduler().get_extra_loop_us());
```

Comment:
Cast?  This is probably a compiler warning on some platform or another.

Reply:
fixed